packages/permission-log-controller/package.json

```diff
@@ -30,7 +30,7 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
-    "@metamask/base-controller": "^3.2.3",
+    "@metamask/base-controller": "^4.0.0",
```

Comment:
This change is here as it was added automatically, but this package is not being released yet, as there are other pending items to complete in order to publish this package.